src/platform/Darwin/DnssdImpl.h

```diff
@@ -27,9 +27,15 @@
 #include <string>
 #include <vector>
 
+constexpr char kLocalDot[] = "local.";
+
+constexpr char kOpenThreadDot[] = "default.service.arpa.";
```

Comment:
Maybe we should update the name to reference SRP instead of OpenThread? In the future we may well be using this on WiFi as well.

Reply:
Yes, please.

Reply:
On WiFi you'll be using a default browsing domain (passing NULL for the domain in the browse) and there won't be a need for a second browse.

Reply:
I used Srp instead of OpenThread